docs/application/dotnet/guides/nui/nui-components/ScrollableBase.md

```diff
@@ -9,30 +9,30 @@ You can use a mouse or finger on the touch screen to drag a ScrollableBase compo
 
 To create a ScrollableBase using various properties, follow these steps:
 
-1. Create ScrollableBase using the default constructor:
+1. Create `ScrollableBase` using the default constructor:
 
     ```csharp
     ScrollableBase scrollableBase = new ScrollableBase();
     ```
 
-2. Set the ScrollableBase property:
+2. Set the `ScrollableBase` property:
 
     ```csharp
-    scrollableBase.Size = new Size(400, 300);
+    scrollableBase.Size = new Size(720, 1280);
```

Comment:
Question. In the ScrollableBase.png and ScrollableBase.gif, 
Only 3 views are shown at once and other views appears when it is scrolled.

Are those image match with updated code?

Also, is there any reason that the height size is 1280, while Views position increase by 200?

Reply:
@patrykkam could you check these comments and respond to them?

Reply:
@patrykka
Gentle reminder, could you please check on the above comments and respond?

Reply:
Sorry for late response, this match example code. But not view on mobile emulator.
Values are changed to reach mobile tizen emulator dafault view.
Can you advise if I should put images from mobile emulator here?

Reply:
@patrykka could you please respond to the above query?

Reply:
You are totaly right, I will update example and ask you to review again.

Reply:
Thank you for you comment, now I understand that description in the docs dont have to match exaclty what is in reference example application.
I restored changes, please review now.

Reply:
What do you mean with "description in the docs dont have to match exaclty what is in reference example application"?
I asked the question because the description and the example didn't seem match.